em-app/src/platform/mod.rs

```diff
@@ -24,4 +28,13 @@ pub(crate) mod nitro;
           any(target_env="gnu",
               target_env="musl",
               target_env = "fortanixvme")))]
-pub(crate) use nitro::get_remote_attestation_parameters;
+pub(crate) use nitro::{get_remote_attestation_parameters, get_remote_attestation_parameters_subject};
+
+pub(crate) fn get_extensions_from_alt_names(alt_names: Option<Vec<Cow<str>>>) -> Option<Vec<(ObjectIdentifier, bool, Vec<u8>)>> {
```

Comment:
This was common code used in the `platform` module, so I opted to make it a function. I do not know if it would be preferable to put it somewhere like a `utils.rs`, but here it is in the module declaration file.